apm-agent-plugins/apm-logging-plugin/apm-jul-plugin/src/main/java/co/elastic/apm/agent/jul/correlation/JulEcsLogCorrelationHelper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.apm.agent.jul.correlation;
+
+import co.elastic.apm.agent.loginstr.correlation.AbstractLogCorrelationHelper;
+import co.elastic.logging.jul.JulMdc;
+
+public class JulEcsLogCorrelationHelper extends AbstractLogCorrelationHelper.DefaultLogCorrelationHelper {
+
+    @Override
+    protected void addToMdc(String key, String value) {
+        JulMdc.getInstance().put(key, value);
```

Comment:
Maybe make these methods static, similar to other MDC/ThreadContext implementations?

```suggestion
        JulMdc.put(key, value);
```